tutorials/gas_compact/openmc.i

```diff
@@ -136,7 +136,7 @@ mdot = 0.011                             # fluid mass flowrate (kg/s)
 
 [Executioner]
   type = Transient
-  num_steps = 4
+  num_steps = 2
```

Comment:
Why was this changed? We should not be decreasing this.

Reply:
My bad, I originally changed this so that the number of steps equaled the number of time steps for the gold files (2, see tests). I did this before I realized `num_steps` could be set from the command line. I can put it back